### DISCUSSION TITLE
Ray as error handler for Wordpress

### DISCUSSION THREAD

**Discussion by jurecuhalev** [spatie/laravel-ray#133] [Ideas]
Is there a way to get ray to be a top level error handler for Wordpress? Doing:

`set_error_handler("ray")` in `functions.php` works great, but it doesn't capture errors that appear in bootstrap wordpress itself. I'm using WP Plugin, so it would be welcome to figure out if ray can be bootstrapped before Wordpress itself.

**Comment by freekmurze** [comment]
I'm not WordPress developer, so I don't know a good way to achieve this. 

I'd accept a PR that improves on this.
